controllers/reviews.controller.js

```diff
@@ -35,8 +35,78 @@ async function getReview (productId) {
     })
 }
 
-async function getReviewByReviewText (reviewText) {
-    return await reviewModel.find({ReviewText: { "$regex": reviewText, "$options": "i"}})
+async function searchThroughEntireReviews (searchText) {
```

Comment:
Can you provide some info on how you tested these changes ?

Reply:
can u get on a call thx

Reply:
As discussed all the methods have been tested